src/connectedk8s/azext_connectedk8s/troubleshoot_diagnoser_job.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: azure-arc-troubleshoot-sa
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: default
+  name: secret-reader
+rules:
+- apiGroups: [""] 
+  resources: ["secrets"]
+  verbs: ["get", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: read-pods
+  namespace: default
+subjects:
+- kind: ServiceAccount
+  name: azure-arc-troubleshoot-sa
+  namespace: default
+roleRef:
+  kind: Role 
+  name: secret-reader 
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: diagnosercontainer
+  namespace: default
+  labels:
+    app: networktest
+spec:
+  template:
+    metadata:
+      labels:
+        app: networktest
+    spec:
+      serviceAccountName: azure-arc-troubleshoot-sa
+      containers: 
+      - name: networktest1
+        image: svacr1.azurecr.io/networktest:7.0
+      restartPolicy: Never
+  backoffLimit: 4
```

Comment:
is there any relevance to having the backoff limit as 4?

Reply:
No specific reasons for that. We can reduce it to 1 or 2.